### PR TITLE
EJDOTNETCORE-4253 Added read me file in create PDF using csharp

### PR DESCRIPTION
**Pull request by sameerkhan001**
Hi Sir,

Added read me file in create pdf document using c#

Regards,
Sameerkhan N

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
-# create-pdf-in-csharp
-This repository has various samples to create PDF file from Word, Presentation, Excel, HTML and images.
+# Create PDF file in C#
+
+The Syncfusion [PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) is a .NET library that allows you to create PDF files programmatically using C#. You can easily create PDF documents from scratch or files in other formats such as Word, Excel, PowerPoint (PPTX), web pages, and images.
```
Comment (ghost) [inline README.md]: The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows users to create PDF files programmatically using C#. Also users can create create PDF documents from scratch or files in other formats such as Word, Excel, PowerPoint (PPTX), web pages, and images.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+| Sample name | Description
+|------|------|
+|[PDF file from scratch](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/master/pdf_from_scratch) | Create a PDF file from scratch With the help of [Syncfusion .NET PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) |
+| [PDF file from a Word template](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/master/Pdf_from_word) | Create a PDF file from a Word template with the help of [Syncfusion .NET Word (DocIO) Library](https://www.syncfusion.com/document-processing/word-framework/net) |
```
Comment (ghost) [inline README.md]: PDF file from Word

File: README.md [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+|[ PDF file from an Excel file](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/master/pdf_from_excel) |Create a PDF file from an Excel file with the help of [ .NET PDF](https://www.syncfusion.com/document-processing/pdf-framework/net) and [Excel library](https://www.syncfusion.com/document-processing/excel-framework/net) |
```
Comment (ghost) [inline README.md]: PDF file from Excel

File: README.md [hunk 4]
```diff
@@ -1,2 +1,43 @@
...
+|[PDF from a PowerPoint](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/master/pdf_from_powerPoint) | Create a PDF from a PowerPoint with the help of Syncfusion .NET [PDF](https://www.syncfusion.com/document-processing/pdf-framework/net) and [PowerPoint Libraries](https://www.syncfusion.com/document-processing/powerpoint-framework/net)|
```
Comment (ghost) [inline README.md]: PDF file from Powerpoint

File: README.md [hunk 5]
```diff
@@ -1,2 +1,43 @@
...
+|[PDF from webpages](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/master/pdf_from_web) |Create a PDF from webpages with the help of [ HTML-to-PDF converter](https://www.syncfusion.com/document-processing/pdf-framework/net/html-to-pdf) |
```
Comment (ghost) [inline README.md]: PDF file from webpages

File: README.md (hunk 1 above)
Comment (sameerkhan001) [reply README.md]: Changed the description

File: README.md (hunk 2 above)
Comment (sameerkhan001) [reply README.md]: Changed

File: README.md (hunk 3 above)
Comment (sameerkhan001) [reply README.md]: changed

File: README.md (hunk 4 above)
Comment (sameerkhan001) [reply README.md]: changed

File: README.md (hunk 5 above)
Comment (sameerkhan001) [reply README.md]: changed

File: README.md [hunk 6]
```diff
@@ -1,2 +1,43 @@
-# create-pdf-in-csharp
-This repository has various samples to create PDF file from Word, Presentation, Excel, HTML and images.
+# Create PDF file in C#
+
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows users to create PDF files programmatically using C#. Also users can create PDF documents from scratch or files in other formats such as Word, Excel, PowerPoint (PPTX), web pages, and images.
+| Sample name | Description
+|------|------|
+|[PDF file from scratch](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/main/pdf_from_scratch) | Create a PDF file from scratch With the help of [Syncfusion .NET PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net) |
```
Comment (ghost) [inline README.md]: With should be "with"

Comment (sameerkhan001) [reply README.md]: resolved

File: README.md [hunk 7]
```diff
@@ -1,2 +1,43 @@
-# create-pdf-in-csharp
-This repository has various samples to create PDF file from Word, Presentation, Excel, HTML and images.
+# Create PDF file in C#
+
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows users to create PDF files programmatically using C#. Also users can create PDF documents from scratch or files in other formats such as Word, Excel, PowerPoint (PPTX), web pages, and images.
+ Sample name | Description
+------|------
+[PDF file from scratch](https://github.com/SyncfusionExamples/create-pdf-in-csharp/blob/main/pdf_from_scratch) | Create a PDF file from scratch with the help of [Syncfusion .NET PDF Library](https://www.syncfusion.com/document-processing/pdf-framework/net)
```
Comment (ghost) [inline README.md]: Add (.) in each sentence

Comment (sameerkhan001) [reply README.md]: added (.) end of the sentence

**Review by praveenHari24 — APPROVED** [review]

**Pull request merged**